src/noahsternberg.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function NoahSternbergComponent() {
+
+    const data = [
+        {id : 1},
+        {projectType : 'Deck building'},
+        {projectLengthInDays : 4},
+        {estimatedLaborCost : 1234},
+        {bidPrice : 1500},
+        {accepted : true}
+    ]
```

Comment:
Try this instead:

const data = [
  { id: 1, projectType: 'Deck Building', projectLengthInDays: 4, estimatedLaborCost: 1234, bidPrice: 1500, accepted: true },
 { id: 2, projectType: 'Deck Building3', projectLengthInDays: 43, estimatedLaborCost: 6564, bidPrice: 600, accepted: false },
];